src/utils/utils.go

```diff
@@ -225,3 +227,26 @@ func SetNoProxyEnv(noProxy string) {
 	os.Setenv("NO_PROXY", noProxy)
 	os.Setenv("no_proxy", noProxy)
 }
+
+func parseField(s string) (string, bool, int) {
```

Comment:
This all seems rather error-prone. To @eranco74 's point in https://github.com/openshift/assisted-installer/pull/80/files#r519886807, can we just index into the struct directly for now rather than adding all this parsing code?

We can generalize when we have at least one other use case.

Reply:
removed